packages/@react-facet/core/src/facet/createFacet.ts

```diff
@@ -12,8 +12,14 @@ export interface FacetOptions<V> {
   startSubscription?: StartSubscription<V>
   equalityCheck?: EqualityCheck<V>
 }
-
-export function createFacet<V>({ initialValue, startSubscription, equalityCheck }: FacetOptions<V>): WritableFacet<V> {
+/**
+ * The low level function to create a Facet, not recommended to be used if you can use any of the react facet hooks to create facets instead (Ex: useFacetState, useFacetWrap)
+ */
+export function createFacet<V>({
+  initialValue,
+  startSubscription,
+  equalityCheck = defaultEqualityCheck,
```

Comment:
We need a unit test to make sure this is the default behaviour now.

Reply:
I just updated the tests now,
I am assuming it might be enough to just remove the passing of `defaultEqualityCheck` in the unit tests that already tests the behavior with that checker?

Or what do you think?

Reply:
Great approach!